### PR TITLE
Add 'ISAR' prefix to compose file

### LINKED ISSUE #244: Can't run isar as container with non-default robot package
**Describe the bug**
When changing the robot type in the docker-compose file, we get the error ` ModuleNotFoundError: No module named 'isar_robot'`.
Somewhere in the code, we are missing a check for environment variables so that it defaults to isar_robot anyways.

**To Reproduce**
Change `ROBOT_REPOSITORY_CLONE_URL`  from "isar-robot" to "isar-turtlebot" and `ROBOT_PACKAGE` from "isar_robot" to "isar_turtlebot".

**Expected behavior**
ISAR should run in a container using the isar turtlebot library.

### PR DESCRIPTION
**Pull request by GodVenn**
Closes #244

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ services:
     build:
       context: .
       args:
-        ROBOT_REPOSITORY_CLONE_URL: https://github.com/equinor/isar-robot.git
+        ROBOT_REPOSITORY_CLONE_URL: https://github.com/equinor/isar-turtlebot.git
```
Comment (MortFred) [inline docker-compose.yml:7]:
```suggestion
        ROBOT_REPOSITORY_CLONE_URL: https://github.com/equinor/isar-robot.git
```
Default value should be isar-robot

File: docker-compose.yml [hunk 2]
```diff
@@ -19,4 +19,4 @@ services:
       - MQTT_PASSWORD
 
       # Selected robot package
-      - ROBOT_PACKAGE=isar_robot
+      - ISAR_ROBOT_PACKAGE=isar_turtlebot
```
Comment (MortFred) [inline docker-compose.yml:22]:
```suggestion
      - ISAR_ROBOT_PACKAGE=isar_robot
```
Default should be isar_robot

File: docker-compose.yml [hunk 3]
```diff
@@ -19,4 +19,4 @@ services:
       - MQTT_PASSWORD
```
Comment (MortFred) [inline docker-compose.yml:19]:
```suggestion
      - ISAR_MQTT_PASSWORD
```

**Review by MortFred — CHANGES_REQUESTED** [review]
See inline comments

**Review by aestene — APPROVED** [review]
LGTM

**Review by MortFred — APPROVED** [review]

**Pull request merged**